ideascube/library/static/library/library.css

```diff
@@ -39,4 +39,145 @@ img.cover {
     font-size: 1em;
-}
\ No newline at end of file
+}
+
+
+
+/* ************************************************* */
+/* ************ SECTION COLORS ********************* */
+/* ************************************************* */
+
+/************* Cartoons *************/
+#id_section > option[value=children-cartoons] {
+  background-color: #e9d460;
+}
+
+.book-detail .children-cartoons {
+  background-color: #e9d460;
+  padding: 0 5px;
+  color: #fff;
+}
+
+.card.children-cartoons,
+.cover.children-cartoons {
+  border: 4px solid #e9d460;
+}
+
+
+/************* Novels *************/
+#id_section > option[value=children-novels],
+#id_section > option[value=adults-novels] {
+  background-color: #4183D7;
+}
+
+.book-detail .children-novels,
+.book-detail .adults-novels {
+  background-color: #4183D7;
+  padding: 0 5px;
+  color: #fff;
+}
+
+.card.children-novels,
+.card.adults-novels,
+.cover.children-novels,
+.cover.adults-novels {
+  border: 4px solid #4183D7;
+}
+
+
+/************* Poetry *************/
+#id_section > option[value=children-poetry],
+#id_section > option[value=adults-poetry] {
+  background-color: #9B59B6;
+}
+
+.book-detail .children-poetry,
+.book-detail .adults-poetry {
+  background-color: #9B59B6;
+  padding: 0 5px;
+  color: #fff;
+}
+
+.card.children-poetry,
+.card.adults-poetry {
+  border: 4px solid #9B59B6;
+}
+
+
+/************* Theatre *************/
+#id_section > option[value=children-theatre],
+#id_section > option[value=adults-theatre] {
+  background-color: #6C7A89;
+}
+
+.book-detail .children-theatre,
+.book-detail .adults-theatre {
+  background-color: #6C7A89;
+  padding: 0 5px;
+  color: #fff;
+}
+
+.card.children-theatre,
+.card.adults-theatre {
+  border: 4px solid #6C7A89;
+}
+
+/************* Tales *************/
+#id_section > option[value=children-tales],
+#id_section > option[value=adults-tales] {
+  background-color: #4DAF7C;
+}
+
+.book-detail .children-tales,
+.book-detail .adults-tales {
+  background-color: #4DAF7C;
+  padding: 0 5px;
+  color: #fff;
+}
+
+.card.children-tales,
+.card.adults-tales,
+.cover.children-tales,
+.cover.adults-tales {
+  border: 4px solid #4DAF7C;
+}
+
+
+/************* Comics *************/
+#id_section > option[value=children-comics],
+#id_section > option[value=adults-comics] {
+  background-color: #D91E18;
+}
+
+.book-detail .children-comics,
+.book-detail .adults-comics {
+  background-color: #D91E18;
+  padding: 0 5px;
+  color: #fff;
+}
+
+.card.children-comics,
+.card.adults-comics {
+  border: 4px solid #D91E18;
+}
+
+
+/************* Myths *************/
+#id_section > option[value=children-myths],
+#id_section > option[value=adults-myths] {
+  background-color: #F27935;
+}
+
+.book-detail .children-myths,
+.book-detail .adults-myths {
+  background-color: #F27935;
+  padding: 0 5px;
+  color: #fff;
+}
+
+.card.children-myths,
+.card.adults-myths,
+.cover.children-myths,
+.cover.adults-myths {
+  border: 4px solid #F27935;
+}
```

Comment:
There are white.

I consider the white as the default and I do not explicit the color here.